client/jetpack-cloud/sections/partner-portal/primary/payment-method-add/index.tsx

```diff
@@ -25,11 +111,80 @@ export default function PaymentMethodAdd(): ReactElement {
 				<CardHeading size={ 36 }>{ translate( 'Payment Method' ) }</CardHeading>
 			</div>
 
-			<Card className="payment-method-add__body">
-				<div className="payment-method-add__body-left"></div>
+			<CheckoutProvider
+				onPaymentComplete={ () =>
+					onPaymentSelectComplete( {
+						successCallback: () => page( '/partner-portal/payment-method/' ),
+						translate,
+						showSuccessMessage,
+					} )
+				}
+				showErrorMessage={ showErrorMessage }
+				showInfoMessage={ showInfoMessage }
+				showSuccessMessage={ showSuccessMessage }
+				paymentMethods={ paymentMethods }
+				paymentProcessors={ {
+					card: ( data ) =>
+						assignNewCardProcessor(
+							{
+								useAsPrimaryPaymentMethod,
+								translate,
+								stripe,
+								stripeConfiguration,
+								dispatch,
+							},
+							data as NewCardSubmitData
+						),
+				} }
+				isLoading={ isStripeLoading }
+				initiallySelectedPaymentMethodId="card"
+			>
+				<Card className="payment-method-add__content">
+					<CardHeading>{ translate( 'Credit card details' ) }</CardHeading>
+
+					{ 0 === paymentMethods.length && <CreditCardLoading /> }
+
+					{ paymentMethods && paymentMethods[ 0 ] && paymentMethods[ 0 ].activeContent }
 
-				<div className="payment-method-add__body-right"></div>
-			</Card>
+					{ storedCards.length > 0 && (
+						<FormLabel className="payment-method-add__all-subscriptions-checkbox-label">
+							<FormInputCheckbox
+								className="payment-method-add__all-subscriptions-checkbox"
+								checked={ useAsPrimaryPaymentMethod }
+								onChange={ () => setUseAsPrimaryPaymentMethod( ( checked ) => ! checked ) }
+								aria-label={ assignAllSubscriptionsText }
+							/>
+							<span>{ assignAllSubscriptionsText }</span>
+						</FormLabel>
+					) }
+
+					<div className="payment-method-add__navigation-buttons">
+						<Button
+							className="payment-method-add__back-button"
+							href="/partner-portal/payment-method/"
+							disabled={ isStripeLoading }
+							onClick={ onGoToPaymentMethods }
+						>
+							{ translate( 'Go back' ) }
+						</Button>
+						<CheckoutSubmitButton className="payment-method-add__submit-button" />
+					</div>
+				</Card>
+			</CheckoutProvider>
 		</Main>
 	);
 }
+
+export default function PaymentMethodAddWrapper(): ReactElement {
```

Comment:
I don't think this is used anywhere? Or is it used later on for something outside this PR? 😄